cmd/tempo/app/modules.go

```diff
@@ -206,12 +231,13 @@ func (t *App) setupModuleManager() error {
 		// Overrides:    nil,
 		// Store:        nil,
 		// MemberlistKV: nil,
+		Frontend:    {Server},
 		Ring:        {Server, MemberlistKV},
 		Distributor: {Ring, Server, Overrides},
 		Ingester:    {Store, Server, Overrides, MemberlistKV},
-		Querier:     {Store, Ring},
+		Querier:     {Store, Ring, Frontend},
```

Comment:
i don't think the querier should require on the frontend.  should it?

Reply:
Oops, you're right, Querier doesn't require Frontend.